lib/twitter.js

```diff
@@ -0,0 +1,81 @@
+const { Rettiwt } = require('rettiwt-api')
+const util = require('./util')
+
+module.exports = class {
+    constructor(users, tweetFilter) {
+        this.users = users
+        this.tweetFilter = tweetFilter
+    }
+
+    async searchAndRetweet({
+        startDate = new Date(),
+        searchInterval = 30 * 1000,
+        retweetDelay = 120 * 1000,
+        isDryRun = false,
+    } = {}) {
+        const rettiwt = new Rettiwt({ apiKey: this.users[0].api_key })
```

Comment:
Don't we want to do this in the constructor?

Reply:
The reason it's there is because, the way it's currently done, there are two cases: the general one for searching (which can use any of the API keys), and the one for retweeting (which is user-specific). Hence why each function instantiates its own at the moment.

Reply:
Gotcha 👍